main.c

```diff
@@ -17,20 +17,25 @@
 #include "error_checks.h"
 #include "baro.h"
 #include "my2c.h"
-#include "lsm303agr.h"
-#include "MPU_6050.h"
+#include "ICM-20948.h"
+#include "ICM-20948_regmap.h"
 #include <xc.h>
 
+#define MAX_BUS_DEAD_TIME_ms 1000
+
 // Set any of these to zero to disable
-#define STATUS_TIME_DIFF_ms 500
-#define BARO_TIME_DIFF_ms 500
-#define IMU_TIME_DIFF_ms 0
-#define PRES_TIME_DIFF_ms 500
-#define TEMP_TIME_DIFF_ms 0
+#define STATUS_TIME_DIFF_ms 500 // 2 Hz
+#define BARO_TIME_DIFF_ms 500 // 2 Hz
+#define IMU_TIME_DIFF_ms 500 // 2 Hz
+#define PRES_OX_CC_TIME_DIFF_ms 500 // 2 Hz
+#define PRES_PNEUMATICS_TIME_DIFF_ms 500 // 2 Hz
+#define TEMP_TIME_DIFF_ms 0 // 2 Hz
```

Comment:
Comment should reflect macro setting in the code, for this case it should be "Disabled"